app/services/app-insights.js

```diff
@@ -2,10 +2,13 @@ const appInsights = require('applicationinsights')
 
 function setup () {
   if (process.env.APPINSIGHTS_INSTRUMENTATIONKEY) {
+    console.log('APPINSIGHTS_INSTRUMENTATIONKEY successfully retrieved!')
```

Comment:
I would move the console line below the `start()` of App Insights and just change it to say something like "App Insights running" because that's all we need to know in the logs.

For the else, I'd either say we don't need to log anything if App Insights isn't running or we change it to highlight that the result of the key not being there is App Insights won't be running.